README.md

```diff
@@ -1 +1,85 @@
-# learning-app
```

Comment:
### README FILE - ADD ALL MISSING back-to-top links


-  Hey, please, kindly reformat your readme using the readme [template](https://github.com/microverseinc/readme-template) by adding all the corresponding `back-to-top` links to improve the navigations in your readme file.

Reply:
### PRESENTATION VIDEO


- Hey, great adding the [video](https://www.loom.com/share/8641547742684cd5a7553ecfbcaae6c1), however, your video should contain a highlight about a piece of the code that you find interesting or proud of making. please, re-record another video to add this missing part of the project requirement.